### PR TITLE
Add s!stoptyping command

### PR DESCRIPTION
**Pull request by LogGame-HU-DEaN**
I don't know if channel.startTyping() is necessary, since the bot is supposed to be typing already.

### CODE REVIEW

**Comment by pedrofracassi** [comment]
Not necessary.

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -204,5 +204,8 @@
     "needMathExpression": "You need to give me a math expression to evaluate",
     "invalidMathExpression": "Invalid math expression",
     "result": "Result: `{{result}}`"
+  },
+  "stoptyping": {
+    "tryingToStop": "Trying to stop writing..."
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: This could be changed to typing

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -204,5 +204,8 @@
     "needMathExpression": "You need to give me a math expression to evaluate",
     "invalidMathExpression": "Invalid math expression",
     "result": "Result: `{{result}}`"
+  },
+  "stoptyping": {
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: You need to add a `commandDescription`

**Comment by bolsomito** [comment]
Suggestion: `channel.stopTyping(true)` forces the client to stop typing completely

**Comment by pedrofracassi** [comment]
Yeah, do that. I knew something like it existed

**Comment by Doges** [comment]
ignore that.

**Comment by Doges** [comment]
ignore too

**Pull request merged**